public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/edit.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { WC_BLOCKS_STORE } from '../../data';
+import { ScheduleGrid } from './schedule-grid';
+import InspectorControls from './inspector-controls';
+import { ICON } from './index';
+
+const blockData = window.WordCampBlocks.sessions || {};
+
+/**
+ * Top-level component for the editing UI for the block.
+ */
+class ScheduleEdit extends Component {
+	/**
+	 * Render the block's editing UI.
+	 *
+	 * @return {Element}
+	 */
+	render() {
+		const { attributes, entities } = this.props;
+
+		return (
+			<Fragment>
+				<ScheduleGrid
+					icon={ ICON }
+					attributes={ attributes }
+					entities={ entities }
+				/>
+
+				<InspectorControls />
```

Comment:
You need to pass `attributes` & `setAttributes` down here, it's causing an error in the InspectorControls now when I try to click any of the toggles/checkboxes.
```suggestion
				<InspectorControls
					attributes={ attributes }
					setAttributes={ setAttributes }
				/>
```

Reply:
It's only meant to be a static scaffolding at this point, but I'm happy to add that.